### PR TITLE
directory table filter click issue

### LINKED ISSUE #545: The directory table filter should handle clicks better
**Describe the bug**
The directory table filter only opens if you click on the text or dropdown

**To Reproduce**
1. Go to https://yahoo.github.io/navi/#/directory/my-data
2. Click between 'All' and the dropdown arrow
3. Dropdown doesn't open/no cursor pointer

**Expected behavior**
The dropdown should open if you click in it.
Make the `selectedFileType` a span that fills the space so it can be clicked

**Screenshots**
Screen Shot 2019-10-03 at 3 22 31 PM

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser chrome
 - Version 77

### PR DESCRIPTION
**Pull request by shubhaemk**
Resolves #545 

## Description

The directory table filter only opens if you click on the text or dropdown

## Proposed Changes
- Wrapping text in `<span>` to increase its width.
- Adding required css.

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/directory/app/styles/navi-directory/components/dir-table-filter.less [hunk 1]
```diff
@@ -12,14 +12,19 @@
...
     cursor: pointer;
-    display: grid;
     font-family: @font-family-thin;
-    grid-template-columns: 1fr 5px;
     margin-left: auto;
     padding: 5px 10px;
     text-align: left;
     user-select: none;
     width: 120px;
+    display: flex;
```
Comment (mattgraham1995) [inline packages/directory/app/styles/navi-directory/components/dir-table-filter.less]: Looks good, but we try to keep our styles in alphabetical order. Could you move display and justify-content up?

File: packages/directory/addon/templates/components/dir-table-filter.hbs [hunk 2]
```diff
@@ -1,7 +1,11 @@
 {{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 {{#basic-dropdown classNames="dir-table-filter" as |dd|}}
   {{#dd.trigger class="dir-table-filter__trigger"}}
-    {{capitalize selectedFileType}}
+    <span 
+      class="dir-table-filter__trigger-type" 
+    >
```
Comment (kevinhinterlong) [inline packages/directory/addon/templates/components/dir-table-filter.hbs]: Can we put this as just `<span class="dir-table-filter__trigger-type">`

File: packages/directory/app/styles/navi-directory/components/dir-table-filter.less (hunk 1 above)
Comment (shubhaemk) [reply packages/directory/app/styles/navi-directory/components/dir-table-filter.less]: Yes sure. I will keep this in mind.

File: packages/directory/addon/templates/components/dir-table-filter.hbs (hunk 2 above)
Comment (shubhaemk) [reply packages/directory/addon/templates/components/dir-table-filter.hbs]: Yes, I forgot to format it after removing onclick and role.

**Pull request merged**